spec/controllers/pages_controller_spec.rb

```diff
@@ -158,26 +158,8 @@
         }
       ]
 
-      stub_request(:get, 'https://suggestions-store.mysociety.org/export/IT/Q1.json')
+      stub_request(:get, "#{ENV.fetch('SUGGESTIONS_STORE_URL').chomp('/')}/export/IT/Q1.json")
         .to_return(body: JSON.generate(suggestions_store_response))
-
-      scheme_data = {
-        results: [
-          {
-            id: 1,
-            name: 'wikidata-persons'
-          },
-          {
-            id: 7,
-            name: 'facebook-persons'
-          }
-        ]
-      }
-
-      stub_request(:get, 'https://id-mapping-store.mysociety.org/scheme')
-        .to_return(body: JSON.pretty_generate(scheme_data))
-      stub_request(:get, "https://id-mapping-store.mysociety.org/identifier/7/175419109847284")
-        .to_return(status: 404, body: '')
```

Comment:
Interesting! I'm _sure_ this was needed when I wrote it, but as you say, seems to work without it, so 👍